src/argilla/client/feedback/training/schemas/base.py

```diff
@@ -499,7 +557,7 @@ def for_sentence_similarity(
                 List[Dict[str, str]],
             ],
         ] = None,
-        label_strategy: Optional[Union[LabelQuestionUnification, RatingQuestionUnification]] = None,
+        label_strategy: Optional[LabelQuestionUnification] = None,
```

Comment:
We decided to support here both `LabelQuestionUnification` and `RatingQuestionUnification` I think?

Reply:
check